Source/Resources/Content/MSGraph/RoleManagement/Directory/RoleDefinitions/Global Administrator.json

```diff
@@ -0,0 +1,169 @@
+{
+  "$description": "Can manage all aspects of Azure AD and Microsoft services that use Azure AD identities.",
+  "inheritsPermissionsFrom": [
+    {
+      "id": "${urn:resource:MSGraph:RoleManagement:Directory:RoleDefinitions/Directory Readers?$id}"
+    }
+  ],
+  "isEnabled": true,
+  "resourceScopes": [
+    "/"
+  ],
+  "rolePermissions": [
+    {
+      "allowedResourceActions": [

```

Comment:
Can you actually edit these, if so do we correctly apply them or are they read only?

Reply:
For built-in resources they only get exported. For custom roles yes.